Sources/SpeziLLM/Views/LLMChatView.swift

```diff
@@ -11,83 +11,105 @@ import SpeziViews
+    @MainActor private var inputDisabled: Bool {
+        llm?.state.representation == .processing
     }
+
     
     public var body: some View {
-        ChatView(
-            $model.context,
-            disableInput: inputDisabled,
-            exportFormat: .pdf
-        )
-            .onChange(of: model.context) { oldValue, newValue in
-                /// Once the user enters a message in the chat, send a request to the local LLM.
-                if oldValue.count != newValue.count,
-                   let lastChat = newValue.last, lastChat.role == .user {
-                    Task {
-                        do {
-                            let stream = try await runner(with: model).generate()
-                            
-                            for try await token in stream {
-                                model.context.append(assistantOutput: token)
+        Group {
+            if let llm {
+                let contextBinding = Binding { llm.context } set: { llm.context = $0 }
+                
+                ChatView(
+                    contextBinding,
+                    disableInput: inputDisabled,
+                    exportFormat: .pdf,
+                    messagePendingAnimation: .automatic
+                )
+                    .onChange(of: llm.context) { oldValue, newValue in
+                        // Once the user enters a message in the chat, send a generation request to the LLM.
+                        if oldValue.count != newValue.count,
+                           let lastChat = newValue.last, lastChat.role == .user {
+                            Task {
+                                do {
+                                    // Trigger an output generation based on the `LLMSession/context`.
+                                    let stream = try await llm.generate()
+                                    
+                                    for try await token in stream {
+                                        llm.context.append(assistantOutput: token)
+                                    }
+                                } catch let error as LLMError {
+                                    llm.state = .error(error: error)
+                                } catch {
+                                    llm.state = .error(error: LLMDefaultError.unknown(error))
+                                }
                             }
-                        } catch let error as LLMError {
-                            model.state = .error(error: error)
-                        } catch {
-                            model.state = .error(error: LLMRunnerError.setupError)
                         }
                     }
-                }
+                    .viewStateAlert(state: llm.state)
+            } else {
+                // Temporarily show an empty `ChatView` until `LLMSession` is instantiated
+                ChatView(
+                    .constant([])
+                )
```

Comment:
Would it make sense to disable the chat view (e.g. all buttons at his point)? Maybe even overlay a progress view if that doesn't look too strange.

Could be that a user tries to interact with the chat while it is loading and then looses all the context.